dev/com.ibm.ws.wssecurity_fat.wsscxf/fat/src/com/ibm/ws/wssecurity/fat/cxf/usernametoken/CxfSSLUNTNonceTests.java

```diff
@@ -164,13 +163,13 @@ public void testCxfUntOldExtFutureTimestampSSLEE7Only() throws Exception {
     }
 
     @Test
-    @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
-    //4/2021
-    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException", "java.util.MissingResourceException", "java.net.MalformedURLException" })
-    public void testCxfUntOldExtFutureTimestampSSLEE8Only() throws Exception {
+    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException", "java.util.MissingResourceException", "java.net.MalformedURLException" },
+                 repeatAction = { JakartaEE9Action.ID })
+    public void testCxfUntOldExtFutureTimestampSSL() throws Exception {
 
         genericTest(
-                    "testCxfUntOldExtFutureTimestampSSLEE8Only",
+                    //"testCxfUntOldExtFutureTimestampSSLEE8Only",
```

Comment:
comment can prob be removed

Reply:
yes, it's missed.  But for consistency with recent cxf saml fat update, I'll be updating the method name with `EE9Only` appended.